lisa/src/main/java/it/unive/lisa/cfg/type/Untyped.java

```diff
@@ -0,0 +1,10 @@
+package it.unive.lisa.cfg.type;
+
+/**
+ * The (singleton) untyped type. 
+ * 
+ * @author <a href="mailto:vincenzo.arceri@unive.it">Vincenzo Arceri</a>
+ */
+public enum Untyped implements Type {
```

Comment:
I think it's better if we turn this into a regular class with a private constructor and a **public static final** singleton field.